packages/eslint-config/next.js

```diff
@@ -6,10 +6,12 @@ const project = resolve(process.cwd(), "tsconfig.json");
 module.exports = {
   extends: [
     "eslint:recommended",
```

Comment:
What's the difference between this file and library.js? And should they be the same?

Reply:
The main difference is that this has the Vercel Next.js eslint rules and is meant to be used in Next apps